fades/parsing.py

```diff
@@ -106,6 +107,32 @@ def _parse_content(fh):
     return deps
 
 
+def _parse_docstring(fh):
+    """Parse the docstrings of a script to find marked dependencies."""
+    requirements = []
+    # get docstring into source file
+    ds_parsed = ast.parse(fh.read())
+    ds_texts = [ds_parsed]
+    ds_texts.extend([node for node in ds_parsed.body if isinstance(node, ast.FunctionDef) or
+                     isinstance(node, ast.ClassDef) or isinstance(node, ast.Module)])
```

Comment:
Do we want to get the docstrings for everything? Or just the module? I vote for "module only"